lightning/src/ln/onchaintx.rs

```diff
@@ -474,6 +474,8 @@ impl<ChanSigner: ChannelKeys> OnchainTxHandler<ChanSigner> {
 
 	/// Lightning security model (i.e being able to redeem/timeout HTLC or penalize coutnerparty onchain) lays on the assumption of claim transactions getting confirmed before timelock expiration
 	/// (CSV or CLTV following cases). In case of high-fee spikes, claim tx may stuck in the mempool, so you need to bump its feerate quickly using Replace-By-Fee or Child-Pay-For-Parent.
+	/// Panics if there are signing errors, because signing operations in reaction to on-chain events
+	/// are not expected to fail, and if they do, we may lose funds.
```

Comment:
Depends, I think this comment is excluding hardware/system failures, which make sense, but what about more subtle cases in the future, like a BOLT13 watchtower client providing garbage for an obfuscated justice transaction ?

Reply:
Wouldn't that be a different code path?  (and does it exist yet?)

Reply:
It doesn't exist yet but a while back I draw a schema for watchtower-support, see : https://github.com/rust-bitcoin/rust-lightning/issues/606#issuecomment-619338323. The first step is to land #642. But it will be the same code path, hitting `generate_claim_tx`. 

Nevermind, we'll update the comment when we have to.